cmd/subcommands/balance.go

```diff
@@ -25,3 +52,10 @@ func init() {
 
 	RootCmd.AddCommand(cmdQuery)
 }
+
+func TestNodeInput(node string) bool{
```

Comment:
lowercase, rename to like, isIPAddr(), because things that start with Test should only be in unit test based functions